README.md

```diff
@@ -61,53 +69,69 @@ This is done using ZGW Consumers, which need to be configured in the Zaken backe
 http://localhost:8080/health
 
 ## Running tests
+
 Run unit tests locally with:
+
 ```
 docker-compose run --rm zaak-gateway python manage.py test
 ```
 
 To run tests for a specific module, add a path:
+
 ```
 docker-compose run --rm zaak-gateway python manage.py test ./apps/cases/tests
 ```
 
 ## Accessing the API documentation
+
 You can access the documentation at:
 http://localhost:8080/api/v1/swagger/
 
 ## Generating an access token
+
 When the LOCAL_DEVELOPMENT_AUTHENTICATION environment variable is set to True, you can gain access easily in the Swagger documentation by executing the /api/v1/oidc-authenticate/ POST request.
 You can use the 'access' token in the response:
 Click on the 'Authorize' button in the top right corner of the page, and enter the given access token.
 This allows you to execute the API endpoints in the page.
 
 ## Generating Mock Data
+
 You can generate mock data easily (from the API swagger environment) by executing the /api/v1/generate-mock/ GET request.
 
 ## Adding pre-commit hooks
+
 You can add pre-commit hooks for checking and cleaning up your changes:
+
 ```
 bash bin/install_pre_commit.sh
 ```
 
 You can also run the following command to ensure all files adhere to coding conventions:
+
 ```
 bash bin/cleanup_pre_commit.sh
 ```
-This will autoformat your code, sort your imports and fix overal problems.
 
+This will autoformat your code, sort your imports and fix overal problems.
 
 ## Coding conventions and style
+
 The project uses [Black](https://github.com/psf/black) for formatting and [Flake8](https://pypi.org/project/flake8/) for linting.
 
 ## Health check
+
 A path is available for checking the health of the running application, and all its connected services.
 The overview of this status can be found on the following path: {application_url}/health
 To improve reliability, the health checks should be expanded for each essential service that is added to the application. For more on how to expand the health checks, read the [Django Healh Check documentation](https://github.com/KristianOellegaard/django-health-check).
+
+## Camunda process deploying
+
+The easieast way to do this is to download the [Camunda Modeler](https://camunda.com/download/modeler/) and use its "Deploy current diagram" function (It is in the toolbar topside al the way to the right next to the play button (start process instance) button. It looks like a horizontal bar with an up arrow on top of it). Makeup a good deployment name and for the REST Endpoint type the url of the base api endpoint. For localhost it is `http://localhost:7000/engine-rest` (NOTE: no end slash) otherwise consult app settings for different environments. DONT FORGET TO CHANGE THE VERSION TAG!!! Camunda will make one for you if you don't but make it nice and correct ;). if you can find the context menu where you can change the version tag: switcht to XML mode and search for the `camunda:versionTag` attribute and change it there.
```

Comment:
Typo: 
- easiest ipv easieast
- switch ipv switch

Version tag: Het stukje over version tag vond ik een beetje onduidelijk. Moet de version tag veranderd worden als je het proces aanpast? Gaat de deploy mis als je de version niet aanpast?

Reply:
Ik krijg bij een deploy nog de volgende errors:

**Bij de ene krijg ik de volgende error te zien:**
ENGINE-09005 Could not parse BPMN process. Errors: 
* attribute 'messageRef' is required | resource zaak_wonen_aanschrijving_verwerken.bpmn | line 35 | column 74
* Invalid 'messageRef': no message with id 'null' found. | resource zaak_wonen_aanschrijving_verwerken.bpmn | line 35 | column 74 [ deploy-error ]

**Bij het andere model:**
ENGINE-09005 Could not parse BPMN process. Errors: 
* Exclusive Gateway 'Gateway_1odl5bp' has outgoing sequence flow 'Flow_0oi0xmk' without condition which is not the default flow. | resource zaak_wonen_vakantieverhuur_huisbezoek_en_debrief.bpmn [ deploy-error ]

Komt dit je bekend voor? Later we er morgen samen even naar kijken :)